browser_extension/js/main.js

```diff
@@ -437,15 +325,15 @@ chrome.storage.sync.get({
         );
 
         //handle clicking channels in the channel box
-        $("div").on("click", ".changechannel", function (e) {
+        $(".changechannel").on("click", ".changechannel", function (e) {
```

Comment:
We have to change $(".changechannel") back to $("div") or it screws up the channel switching